profiles/models/request_notification.py

```diff
@@ -40,8 +40,10 @@ def is_authorized(self, request):
 
     def when_render(self, request):
         from service_catalog.api.serializers import RequestSerializer
+        from service_catalog.api.serializers import InstanceSerializer
         context = {
-            "request": RequestSerializer(request).data
+            "request": RequestSerializer(request).data,
+            "instance": InstanceSerializer(request.instance).data
```

Comment:
The instance serializer is in the request serializer, if I'm not wrong

Reply:
We have two types of notifications, one for instance that contains the squest instance in the context and one for request (this one) that contains only the squest request in his context. The goal here is to be able to use an InstanceNotificationFilter in a RequestNotificationFilter. If you don't agree, remove it :)

Reply:
We have two types of notifications, one for instance that contains the squest instance in the context and one for request (this one) that contains only the squest request in his context. The goal here is to be able to use an InstanceNotificationFilter in a RequestNotificationFilter. If you don't agree, remove it :)

Reply:
We have two types of notifications, one for instance that contains the squest instance in the context and one for request (this one) that contains only the squest request in his context. The goal here is to be able to use an InstanceNotificationFilter in a RequestNotificationFilter. If you don't agree, remove it :)